packages/core/src/Synchronizer.ts

```diff
@@ -589,6 +594,36 @@ export class Synchronizer extends EventEmitter {
         return preimages
     }
 
+    async genSealedEpochProof(
```

Comment:
We should include a note in the docs that this function shouldn't be used in production. Instead use something like [epoch-sealer](https://github.com/unirep/epoch-sealer).

Reply:
I think attesters can still run there epoch-sealer so that they don't have to rely on a single epoch-sealer

Reply:
They don't need to rely on the epoch-sealer that we're running. They can run their own copy of epoch-sealer and have it only seal epochs from their attester.